bundles/org.openhab.binding.speedtest/README.md

```diff
@@ -0,0 +1,48 @@
+# speedtest Binding
+
+This binding simplifies the addition of network speed tests from your openHAB instance.   Simply put, it's a wrapper around Ookla's SpeedTest ( https://www.speedtest.net/apps/cli ).  
+
+You MUST have the Ookla cli application installed on your machine prior to using this.
+
+## Supported Things
+
+Speedtest thing.   
+
+## Binding Configuration
+
+For this binding to work, you MUST install Ookla's speedtest, this will not work with other versions of "speedtest-cli" or packaged speedtest varieties.   
+
+To install Ookla's version of speedtest, head to https://www.speedtest.net/apps/cli and follow the instructions for your Operating System.   
+
+Linux based systems will assume a default install location of /usr/bin/speedtest .  If your environment is different, just enter the path in the Thing config.  
+
+## Thing Configuration
+| config option  |  description                  |
+|----------|------------------------------|
+| Refresh Rate  | This will change the refresh rate(or how often) the binding checks the speed  |
+| Speedtest Path  | This is the full URL to the speedtest executable ** |
+
+The refresh rate can also be set to "Do not test automtically" which will never test.   This can be used if you want to use the trigger_test to test via rules, or an item instead.
```

Comment:
```suggestion
The refresh rate can also be set to "Do not test automatically" which will never test.
This can be used if you want to use the trigger_test to test via rules, or an item instead.
```
same here and there is also a spelling mistake fixed in the suggestion.

Reply:
done